gcc.symbols.imp

```diff
@@ -12,9 +12,10 @@
 # equal.  The visibility on the symbol-name is ignored; by convention
 # we always set it to private.
 [
-  { symbol: [ "blksize_t", private, "<sys/types.h>", public ] },
-  { symbol: [ "blkcnt_t", private, "<sys/stat.h>", public ] },
+  { symbol: [ "aiocb", private, "<aio.h>", public ] },
   { symbol: [ "blkcnt_t", private, "<sys/types.h>", public ] },
+  { symbol: [ "blkcnt_t", private, "<sys/stat.h>", public ] },
```

Comment:
The `blkcnt_t` and `blksize_t` changes should probably go in their own commit(s)

Reply:
Hmm, I think I accidentally commited those changes together.  I seem to recall that I had done it separately...

Fixed.  I did both reorderings in a single commit previous to the aiocb commit.  Breaking into a commit for blkcnt_t and another for blksize_t felt like too much, and a bit artificial to separate.